Examples/ObjectiveC/LocationPrivacyExample.m

```diff
@@ -0,0 +1,51 @@
+#import "LocationPrivacyExample.h"
+@import Mapbox;
+
+NSString *const MBXExampleLocationPrivacy = @"LocationPrivacyExample";
+
+@interface LocationPrivacyExample () <MGLMapViewDelegate>
+@end
+
+@implementation LocationPrivacyExample
+
+- (void)viewDidLoad {
+    [super viewDidLoad];
+
+    MGLMapView *mapView = [[MGLMapView alloc] initWithFrame:self.view.bounds];
+    mapView.autoresizingMask = UIViewAutoresizingFlexibleWidth | UIViewAutoresizingFlexibleHeight;
+    mapView.delegate = self;
+
+    // Enable heading tracking mode so that the puck will appear.
+    mapView.userTrackingMode = MGLUserTrackingModeFollow;
+
+    [self.view addSubview:mapView];
+}
+
+/**
+    In order to enable the alert that requests temporary precise location, please add the following key to your info.plist
+    Privacy - Location Temporary Usage Description Dictionary
+
+    You must then add MGLAccuracyAuthorizationDescription as key in the Privacy - Location Temporary Usage Description Dictionary
+ */
+- (void)mapView:(MGLMapView *)mapView didChangeLocationManagerAuthorization:(id<MGLLocationManager>)manager {
+    if (@available(iOS 14, *)) {
+        if (manager.accuracyAuthorization == CLAccuracyAuthorizationReducedAccuracy) {
```

Comment:
Won't the system pop up appear if we just requestTemporary access?

Reply:
I talked to @fabian-guerra about this. When adding plist keys, that temporary access is automatically handled by us. The alert that is added is a recommendation of how they can handle with the user